todovite/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Everything inside todovite should be in root folder, we don't need todovite folder